src/quo2/components/info/network_breakdown.cljs

```diff
@@ -0,0 +1,77 @@
+(ns quo2.components.info.network-breakdown
+  (:require [quo2.components.icon :as icons]
+            [quo.react-native :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.markdown.text :as text]
+            [status-im.i18n.i18n :as i18n]
+            [clojure.string :as string]))
+
+(defn network-breakdown
+  [{:keys [top-value network-conversions]}]
+  [rn/view {:style {:background-color (colors/theme-colors
+                                       colors/white
+                                       colors/neutral-95)
+                    :padding-horizontal 40
+                    :padding-top 24
+                    :border-radius 16
+                    :height 126
+                    :width 390
+                    :overflow :hidden}}
+   [rn/view {:style {:border-bottom-width 1
+                     :border-bottom-color (colors/theme-colors
+                                           colors/neutral-20
+                                           colors/neutral-70)
+                     :padding-vertical    6}}
+    [text/text {:weight :medium
+                :style  {:color     (colors/theme-colors
+                                     colors/black
+                                     colors/white)
+                         :font-size 19}}
+     (str top-value)]]
+   [rn/scroll-view {:horizontal true
+                    :style {:text-align :center
+                            :margin-top 6
+                            :width 340}}
+    (let [last-item-idx (-> network-conversions
+                            count
+                            dec)]
+      (map-indexed
+       (fn [idx {:keys [conversion network icon]}]
+         [rn/view {:style {:flex-direction :row}
+                   :key   idx}
```

Comment:
More like a FYI than anything else.

It's not recommended in React(Native) apps to use indexes as keys, unless we're confident the collection won't ever be reordered or filtered (due to a re-render pass). Otherwise this causes performance degradation.

Even though the more robust approach should probably avoid indexes as keys in the `network-breakdown` component, I actually think it's fine for this case.

Reply:
Thanks Icaro